src/index.js

```diff
@@ -111,6 +111,16 @@ export default class Form extends Component {
       return
     }
 
+    if (!equals(this.props.errors, nextErrors)) {
+      const propsErrorsKeys = Object.keys(this.props.errors || {})
```

Comment:
Use `keys` from Ramda for browser compatibility?

```suggestion
      const propsErrorsKeys = keys(this.props.errors || {})
```